docs/notes/feature-livecode_builder.md

```diff
@@ -0,0 +1,35 @@
+# LiveCode Builder
+
+## LiveCode Builder Language
+
+LiveCode Builder is a variant of the current LiveCode scripting language (LiveCode Script) which has been designed for 'systems' building. It is statically compiled with optional static typing and direct foreign code interconnect (allowing easy access to APIs written in other languages). Tee compiled bytecode can then be packaged together with any required resources (icons, documentation, images, etc) into a .lce extension package.
+
+Unlike most languages, LiveCode Builder (LCB) has been designed around the idea of extensible syntax. Indeed, the core language is very small - comprising declarations and control structures - with the majority of the language syntax and functionality being defined in modules.
+
+**Note:** It is an eventual aim that control structures will also be extensible, however this is not the case in the current incarnation).
+
+The syntax will be familiar to anyone familiar with LiveCode Script, however LiveCode Builder is a great deal more strict - the reason being it is intended that it will eventually be compilable to machine code with the performance and efficiency you'd expect from any 'traditional' programming language. Indeed, over time we hope to move the majority of implementation of the whole LiveCode system over to being written in LiveCode Builder.
+
+**Note:** One of the principal differences is that type conversion is strict - there is no automatic conversion between different types such as between number and string. Such conversion must be explicitly specified using syntax (currently this is using things like *... parsed as number* and *... formatted as string*.
```

Comment:
currently this is "done"? using "syntax"? like *...* 

That might read slightly more fluidly?